packages/compass-web/sandbox/stored-connections-history.tsx

```diff
@@ -0,0 +1,98 @@
+import { useCallback, useState, useEffect } from 'react';
+import type { ConnectionInfo } from '@mongodb-js/connection-storage/renderer';
+import { Label } from '@mongodb-js/compass-components';
+import { redactConnectionString } from 'mongodb-connection-string-url';
+import React from 'react';
+import { ConnectionsList } from './connections-list';
+
+function getHistory(): ConnectionInfo[] {
+  try {
+    const b64Str = localStorage.getItem('CONNECTIONS_HISTORY');
+    if (!b64Str) {
+      return [];
+    }
+    const binStr = window.atob(b64Str);
+    const bytes = Uint8Array.from(binStr, (v) => v.codePointAt(0) ?? 0);
+    const str = new TextDecoder().decode(bytes);
+    return JSON.parse(str);
+  } catch (err) {
+    return [];
+  }
+}
+export function saveHistory(history: any) {
```

Comment:
just a silly question, what is this for, do we have a connection history?

Reply:
This is just for local dev convenience, we store connections in local storage so that you can reuse them when running this locally, this is sandbox code, it was here from the beginning

Reply:
ahh right cause this still doesn't have multiple connections, and it can still be useful later to auto-connect, makes sense

Reply:
Yep, I think as soon as we are fully in multiconnection world, all this stuff will go away and the roots of two apps will be way more aligned